contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,134 @@
+
+> **Growth engineers vs growth hackers:** Growth hackers also aim to drive growth but focus on more non-product, non-technical solutions. Growth engineering, for some teams, might include this, but for us, the role focuses on technical work across products.
+
+### Product and design knowledge
+
+Second, the other part of creating and implementing ideas is **product and design knowledge**. Because a growth team works across the stack and teams, they are often "lone wolves." They need the ability to fend for themselves in product and design decisions. They don’t have to be Da Vinci, but this knowledge helps them create successful features.
+
+As [Porter](https://www.workatastartup.com/jobs/46166) says:
+
+> We're looking for a product engineer with strong design chops who is fluent in React/Typescript and CSS. Simply put, we want someone who will take Porter's usability and UX/visual polish to the next level. You will be involved in all feature discussions and will shape the trajectory of our product.
+
+### Business understanding
+
+Third, to create growth ideas based on solid strategies, growth engineers must have an **understanding of business**. This includes areas like marketing and finance. Without business knowledge, they can’t effectively make decisions about growth metrics related to those areas. They also need this knowledge to collaborate well with non-technical teams.
+
+### Data literacy
+
+Since growth engineers run many experiments, they must be able to craft hypotheses and evaluate results. They should know what factors are relevant, how to calculate significance, and what data to care about in the first place.
+
+As [Terra](https://www.workatastartup.com/jobs/59415) says:
+
+> The ideal candidate will have a deep understanding of data analytics, website optimization, user acquisition, and user engagement strategies. You will be responsible for developing and executing strategies to grow web traffic and engagement with our target customers. This includes optimizing websites, analyzing user data, and making recommendations for changes to the product that will help us reach our goals.
+
+If all this sounds like a unicorn, you aren’t wrong. This is again why we find former technical founders as a good fit for this role. They have the business and product knowledge to develop growth ideas and strategies, the practical technical knowledge to implement them, as well as a familiarity with the data a product produces.
+
+![Growth engineers' power](../images/blog/what-is-a-growth-engineer/thanos.jpg)
+
+> **Growth engineers vs product engineers:** Both build new features, are opinionated, experiment, and take ownership. The difference is their focus. Product engineers focus on building products for specific users. Growth engineers focus on projects that drive growth. This could be a feature enabling product teams, a service supporting marketing, or an optimization to a cross-product flow. They are less constrained to a specific product.
```

Comment:
```suggestion
> **Growth engineers vs product engineers:** Both build new features, are opinionated, experiment, and take ownership. The difference is their focus. [Product engineers](/blog/what-is-a-product-engineer) focus on building products for specific users. Growth engineers focus on projects that drive growth. This could be a feature enabling product teams, a service supporting marketing, or an optimization to a cross-product flow. They are less constrained to a specific product.
```